fenix/app/src/main/java/org/mozilla/fenix/settings/SupportUtils.kt

```diff
@@ -38,6 +38,7 @@ object SupportUtils {
     const val GOOGLE_US_URL = "https://www.google.com/webhp?client=firefox-b-1-m&channel=ts"
     const val GOOGLE_XX_URL = "https://www.google.com/webhp?client=firefox-b-m&channel=ts"
     const val WHATS_NEW_URL = "https://www.mozilla.org/firefox/android/notes"
+    const val LEARN_FX_SUGGEST = "https://support.mozilla.org/en-US/kb/search-suggestions-firefox"
```

Comment:
Let's get a follow-up bug on file to link to the final SUMO page, once https://bugzilla.mozilla.org/show_bug.cgi?id=1855091 is ready.

Reply:
We might also want to use `getSumoURLForTopic` or `getGenericSumoURLForTopic` here, instead of hard-coding the `en-US` URL (even though it is US-only to start with).

Let's see what the mobile folks think!

Reply:
oooh i missed `getGenericSumoURLForTopic` when i was first digging around. Makes sense to me, but still curious what mobile folks think.

Reply:
I'm unaware of how SUMO handles translations, but it looks like replacing the locale in the URL above will link to translated pages, for example here the link above with the locale switched to Spanish: https://support.mozilla.org/es/kb/search-suggestions-firefox. 

Not sure if there is some auto-translate magic happening or if there are already existing translated pages for that article. I would say that using `getGenericSumoURLForTopic` looks the most future proof, but I am not sure what happens if we generate a link for a locale that does not have a translation available.